### PR TITLE
API 500 Support for Enclosure Group

### LINKED ISSUE #297: Support for Enclosure Group - API500
### Scenario/Intent
Add support to Enclosure Group on the API500.

### Environment Details
 - **oneview Cookbook:** Version: 2.3.0
 - **OneView Version:** 3.10
 - **chef-client Version:** 12.0+
 - **platform:** Any

### Steps to Reproduce
Try to use the resource Enclosure Group on the API 500.

### Expected Result
This resource supports API 500.

### Actual Result
This resource does not support API 500.

### PR DESCRIPTION
**Pull request by aalexmonteiro**
### Description
Adding Enclosure Group for HPE OneView API500 support.

### Issues Resolved
#297 

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
  - [ ] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [x] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -484,7 +484,8 @@ oneview_enclosure_group 'EnclosureGroup_1' do
   client <my_client>
   data <resource_data>
   logical_interconnect_groups ['LIG_name1', { name: 'LIG_name2', enclosureIndex: 1 }]
-  action [:create, :create_if_missing, :delete]
+  script # String - The script to be executed
```
Comment (tmiotto) [inline README.md]: You just pass the script property alone?

**Review by tmiotto — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aalexmonteiro) [reply README.md]: @tmiotto Yeah! It's just a string.

Comment (tmiotto) [reply README.md]: Yeah, it is a String... Where is the String?

Comment (aalexmonteiro) [reply README.md]: @tmiotto This way, is it better?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ## 2.3.1 (Unreleased)
```
Comment (tmiotto) [inline CHANGELOG.md]: If you are adding new resources the version should be 2.4.0, shouldn't it?

File: examples/enclosure_group.rb [hunk 3]
```diff
@@ -25,6 +25,13 @@
   action :create
 end
 
+# The set_script action is only available for C7000.
```
Comment (tmiotto) [inline examples/enclosure_group.rb:28]: You can use the api_variant property in the example if you want to make it more clear

**Review by tmiotto — APPROVED** [review]

**Pull request merged**